modules/renter.go

```diff
@@ -219,6 +219,7 @@ type RenterContract struct {
 	ID            types.FileContractID
 	HostPublicKey types.SiaPublicKey
 	Transaction   types.Transaction
+	SecretKey     crypto.SecretKey
```

Comment:
This shouldn't be necessary

Reply:
It is used in `TestIntegrationMetadata` to pass it to `GetMetadata`